tests/test_office_structure/test_delete_level.py

```diff
@@ -0,0 +1,23 @@
+from tests.base import BaseTestCase, CommonTestCases
+from fixtures.office_structure.delete_level_fixtures import (
+  delete_level_fixtures, delete_level_response,
+  delete_level_fixtures_non_existant_id
+)
+
+
+class TestDeleteLevel(BaseTestCase):
+    def test_delete_level_works(self):
+        """Test whether admin can delete level"""
```

Comment:
We are not testing whether admin can delete level, we are testing that admin can delete a level. Same to any other tests, avoid using `whether`

Reply:
This has been rectified @Bonifase